hardware_interface/include/joint_limits/joint_limits_rosparam.hpp

```diff
@@ -14,17 +14,60 @@
 
 /// \author Adolfo Rodriguez Tsouroukdissian
 
-#ifndef JOINT_LIMITS_INTERFACE__JOINT_LIMITS_ROSPARAM_HPP_
-#define JOINT_LIMITS_INTERFACE__JOINT_LIMITS_ROSPARAM_HPP_
-
-#include <joint_limits_interface/joint_limits.hpp>
-
-#include <rclcpp/rclcpp.hpp>
+#ifndef JOINT_LIMITS__JOINT_LIMITS_ROSPARAM_HPP_
+#define JOINT_LIMITS__JOINT_LIMITS_ROSPARAM_HPP_
 
+#include <limits>
 #include <string>
 
-namespace joint_limits_interface
+#include "joint_limits/joint_limits.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+namespace joint_limits
+{
+inline bool declare_parameters(const std::string & joint_name, const rclcpp::Node::SharedPtr & node)
 {
+  const std::string param_base_name = "joint_limits." + joint_name;
```

Comment:
I am using this in the moveit2's pilz_industrial_motion_planner ([WIP port to ROS 2](https://github.com/henningkayser/moveit2/pull/10/files)) and I think it might be beneficial to include into the API the possibility to load joint_limits in a specific namespace i.e.
```suggestion
inline bool declare_parameters(const std::string& joint_name, const std::string& param_ns, const rclcpp::Node::SharedPtr& node)
{
  const std::string param_base_name = param_ns + "joint_limits." + joint_name;
```
What do you think?

Reply:
This should not be needed because we use parameters namespace from the node.